features/FooterUpdate.feature

```diff
@@ -0,0 +1,66 @@
+@owner:derek
+@author:lilla
+Feature: Docs Footer Update
+
+    New responsive design and links for the footer on the docs pages.
+
+    Background: A user arrives on the SaaSquatch help center page
+        Given the user stays on the 'https://docs.saasquatch.com/' pages
+        Then the footer categories are visible
+
+
+    @motivating
+    Scenario: Categories and links for the footer
+        Given the user looks at the footer
+        Then the following categories and links are visible
+        And the user can click on them
```

Comment:
We try to stay away from specifying what a user can do, instead specify what the user `does`